lib/adapters/looks-same.js

```diff
@@ -7,6 +7,12 @@ var objectAssign = require('object-assign');
 *
 * @implements {Differ}
 * @class
+*
+* @param {Object} [userOptions] - Options for LooksSame
+* @param {Boolean} [userOptions.strict] - Verifies every pixels
```

Comment:
The description is misleading. `looks-same` checks all the pixels but you can control the mismatch tolerance. You should explain that there's a relation between `strict` and `tolerance`, mainly that you can't have both together.

You should also add a link to the `looks-same` docs.